requirements.txt

```diff
@@ -18,6 +18,10 @@ webrtcvad
 # Requirements for importers
 sox
 bs4
+pandas
 requests
 librosa
 soundfile
+
+# Requirements for play.py
+simpleaudio
```

Comment:
Looks like this package requires system-wide dependencies (see Taskcluster failures). Might want to remove the dependency here and check for an ImportError in play.py instead.